### ISSUE TITLE
Python code generation behaviour when dealing with valid XML that represents an invalid WSDL

### ISSUE THREAD

**Issue by techdragon** [soapteam/soapfish#105]
I decided to add some extra test cases by using the [W3C WSDL 2.0 Test Suite](https://dev.w3.org/2002/ws/desc/test-suite/index.html) and I've quickly run into a rather fundamental question. 

What should we do when asked to generate code for an invalid WSDL? I'll quote the W3C doc to help explain.
> We are mainly concerned with test documents that violate rules that cannot be expressed in the XML schema. Such rules must be checked by validators, for example the Java WSDL 2.0 validator that is part of the Apache Woden project

What should soapfish do when asked to generate code for such an invalid document? 

I'm in favour of a permissive default mode, which is what the current behaviour seems to be (4 out of 4 invalid WSDL documents I have tested so far have generated python code), and adding an optional strict mode as a feature later on that will warn or refuse to handle invalid WSDL documents at the users discretion.

Thoughts?

**Comment by pope1ni** [comment]
Agreed. Permissive is best as there are too many bits unimplemented. We could try to implement a strict mode, but we'd have to blow up for everything that isn't supported yet.
